x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/kpis/kpi.tsx

```diff
@@ -1,70 +0,0 @@
-/*
```

Comment:
This file is a duplicate of [x-pack/plugins/observability_solution/infra/public/pages/metrics/hosts/components/kpis/kpi_charts.tsx](https://github.com/elastic/kibana/pull/180392/files#diff-23ccfc3412d969670195715f6ca5fcf6a3383a883e21d565d89532b96024662b) and it wasn't used anywhere so I removed it